picoquictest/tls_api_test.c

```diff
@@ -8616,6 +8619,9 @@ int ddos_amplification_test_one(int use_0rtt)
             picoquic_delete_cnx(test_ctx->cnx_server);
             /* Delete the old client connection */
             picoquic_delete_cnx(test_ctx->cnx_client);
+            for (size_t i = 0; i < test_ctx->nb_test_streams; i++) {
+                 test_api_delete_test_stream(&test_ctx->test_stream[i]);
+            }
```

Comment:
Same as above

Reply:
Yes. And that's hiding the missing code in the test context deletion.